dev_apps/music-nga/views/albums/view.js

```diff
@@ -56,6 +57,14 @@ AlbumsView.prototype.getAlbums = function() {
     .then(albums => clean(albums));
 };
 
+AlbumsView.prototype.search = function(query) {
+  return this.fetch('/api/search/album/' + query).then((response) => {
```

Comment:
nit: I've been generally trying to wrap the `fetch()` calls in a method on the view for better readability.